lib/src/widgets/buttons.dart

```diff
@@ -392,3 +396,85 @@ class _CardButtonState extends State<CardButton> {
     }
   }
 }
+
+/// Button to add onTap on long pressing
+///
+/// ### Note
+/// Widgets with `tooltip` don't handle onLongPress
+/// Child's onTap will have priority
+class LongPressButton extends StatefulWidget {
```

Comment:
While I think the implementation is correct, the name of the class and the doc are not relevant.

I'd call this button `RepeatButton` because that's what he does, calling repeatedly a function on hold, triggered after a long press.

The doc comment should explain that and the fact the button handles the `onTap` too.

It should mention too that the visual of the button is delegated to the child, this one is just a repeat wrapper.